drivers/include/periph/flashpage.h

```diff
@@ -106,6 +122,26 @@ static inline int flashpage_page(void *addr)
  */
 void flashpage_write(int page, void *data);
 
+/**
+ * @brief   Write the given address with the given data and length
```

Comment:
IMHO here is still room for improvement. So let me propose something:
`Write any number of data bytes to a given location in the flash memory`